server/server.js

```diff
@@ -3,7 +3,7 @@ const app = express();
 const dotenv = require("dotenv");
 const cors = require("cors");
 const db = require("./utils/dbConfig");
-const port = process.env.NODE_PORT;
+const port = process.env.PORT || 4000;
```

Comment:
can you change this back to NODE_PORT?

Reply:
once you finish this i can merge your code

Reply:
I have tried with NODE_PORT initially, but heroku is not understanding it so I changed to PORT as veni also said that

Reply:
In that case, write a condition to use PORT only for .env.production
but for local/dev you should use .env.NODE_PORT so that it doesn't break for others.

NOTE: Don't just change things just like that which could actually break things for others. Ask before making such breaking changes.

Reply:
Is this done?